src/mappings/mappingHandlers.ts

```diff
@@ -136,7 +146,6 @@ export async function handleEvent(event: CosmosEvent): Promise<void> {
   logger.info(`[handleEvent] (tx ${event.tx.hash}): indexing event ${event.idx + 1} / ${event.tx.tx.events.length}`)
   logger.debug(`[handleEvent] (event.event): ${JSON.stringify(event.event, null, 2)}`)
   logger.debug(`[handleEvent] (event.log): ${JSON.stringify(event.log, null, 2)}`)
-
   // NB: sanitize attribute values (may contain non-text characters)
```

Comment:
nit: was there a reason to delete this blank line back?

```suggestion

  // NB: sanitize attribute values (may contain non-text characters)
```